pkg/util/env_test.go

```diff
@@ -0,0 +1,62 @@
+package util
+
+import (
+	"testing"
+	"os"
+	"github.com/magiconair/properties/assert"
+)
+
+func TestLookEnv(t *testing.T) {
+	// set required environment
+	os.Setenv("_MY_PRESENT_TEST_KEY_", "value1")
+	os.Unsetenv("_MY_MISSING_TEST_KEY_") // for safety
+
+	cases := map[string]struct {
+		key         ENVKey
+		expectValue string
+	}{
+		"Missing env variable": {
+			key:         "_MY_MISSING_TEST_KEY_",
+			expectValue: "false",
+		},
+		"Present env variable": {
+			key:         "_MY_PRESENT_TEST_KEY_",
+			expectValue: "value1",
+		},
+	}
```

Comment:
Add a case where key is present without any value ?

Reply:
done